cli.js

```diff
@@ -9,6 +9,7 @@ const SparkPost = require('sparkpost');
 const sparkpost = new SparkPost(SPARKPOST_API_KEY);
 const subbable = require('./lib/subbable');
 const defaultFromSparkPost = require('./lib/default-from-sparkpost');
+const CLI_NAME = 'sparkpost';
```

Comment:
Would `path.basename(process.argv[1])` be a better fit, in case you or someone else wants to rename the command?